.github/workflows/compile_nuttx.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: px4io/px4-dev-nuttx-bionic:2020-04-01
+    container: px4io/px4-dev-nuttx-bionic:2020-08-14
```

Comment:
Can we please change the tags for all the build jobs instead? Otherwise it becomes more difficult to update them further if they are different from each other.

Reply:
done in 867578d